tv_apps/tv-epg/js/epg.js

```diff
@@ -49,6 +46,10 @@
       this.visibleTimeOffset - this.visibleTimeSize,
       this.timelineUnit
     );
+
+    this.clock = new Clock();
```

Comment:
What's the difference that you move clock code here? It looks that we don't need to do it.